response_operations_ui/templates/collection_exercise/ce-ci-section.html

```diff
@@ -8,14 +8,28 @@
           section. "
       })
     }}
+  {% if survey["surveyMode"] == "EQ_AND_SEFT" %}
+      {% set tblHeaders =
+        [
+                  {
+                  "value": "",
+                  },
+                  { 
+                        "value":'<em>' + "Total CI " + '(' + ci_table_context["total_ci_count"] + ')' '</em>',
```

Comment:
This can be rationalized to ```"<em>Total CI (" + ci_table_context["total_ci_count"] + ")</em>" ```

Reply:
Updated the code to reflect those changes